model/src/main/java/org/projectnessie/model/EntriesResponse.java

```diff
@@ -50,8 +53,13 @@ static ImmutableEntry.Builder builder() {
     @Value.Parameter(order = 1)
     ContentKey getName();
 
-    static Entry entry(ContentKey name, Content.Type type) {
-      return ImmutableEntry.of(name, type);
+    @JsonView(Views.V2.class)
+    @Value.Parameter(order = 3)
+    @Nullable // for V1 backwards compatibility
+    String getContentId();
+
+    static Entry entry(ContentKey name, Content.Type type, String contentId) {
```

Comment:
Would it make sense to keep the old override in case some old client rely on it? The new attribute is nullable, so we can maintain a compatible implementation of the old `entry` method.

Reply:
not sure whether clients have a reason to call this method, but good point.